README.md

```diff
@@ -76,49 +76,311 @@ Crane is composed of the following components:
 
 ## Getting Started
 
-### Prerequisites
+### Installation
+
+**Prerequisites**
 
 - Kubernetes 1.18+
 - Helm 3.1.0
 
-### Installation
-
-#### Installing prometheus components with helm chart
+**All-In-One Installation**
 
 > Note:
-> If you already deployed prometheus, prometheus-node-exporter, then you can skip this step.
+> If you already deployed prometheus, grafana, or you want to customize it then you can refer to [Customize Installation](#customize-installation).
+
+**Helm Installation**
+Please refer to Helm's [documentation](https://helm.sh/docs/intro/install/) for installation.
+
+**Installing prometheus and grafana with helm chart**
 
-Export the following env if you want to use default settings, or specify customized value if you want to customize the installation.
+Crane use prometheus to be the default metric provider. Using following command to install prometheus components: prometheus-server, node-exporter, kube-state-metrics.
 
 ```console
-export NAMESPACE=monitoring
-export RELEASE_NAME=myprometheus
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm install prometheus -n crane-system --set pushgateway.enabled=false --set alertmanager.enabled=false --set server.persistentVolume.enabled=false -f https://raw.githubusercontent.com/gocrane/helm-charts/main/integration/prometheus/override_values.yaml --create-namespace  prometheus-community/prometheus
 ```
 
-Crane use prometheus to be the default metric provider. Using following command to install prometheus components.
+Fadvisor use grafana to present cost estimates. Using following command to install a grafana.
 
 ```console
-helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-helm repo update
+helm repo add grafana https://grafana.github.io/helm-charts
+helm install grafana -f https://raw.githubusercontent.com/gocrane/helm-charts/main/integration/grafana/override_values.yaml -n crane-system --create-namespace grafana/grafana
```

Comment:
is there a flag to set grafana service as NodePort or LoadBalancer by default?

Reply:
Will address it in later pr